.ci/travis/travis.sh

```diff
@@ -264,6 +264,50 @@ no-error-test-sbe)
   removeFolderWithProtectedFiles simple-binary-encoding
   ;;
 
+no-error-xwiki)
+  # TODO: put back when https://github.com/checkstyle/checkstyle/issues/9236 is fixed
+  # CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
+  #                   --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
+  CS_POM_VERSION=8.39
+  echo version:$CS_POM_VERSION
+  mvn -e clean install -Pno-validations
+  mkdir -p .ci-temp/
+  cd .ci-temp/
+  git clone --depth 1 https://github.com/xwiki/xwiki-commons.git
+  cd xwiki-commons
+  # Build custom Checkstyle rules
+  mvn -e -f xwiki-commons-tools/xwiki-commons-tool-verification-resources/pom.xml \
+    install -DskipTests -Dcheckstyle.version=${CS_POM_VERSION}
+  # Validate xwiki-commons
+  mvn -e checkstyle:check@default -Dcheckstyle.version=${CS_POM_VERSION}
```

Comment:
as we start to validate almost whole xwiki
please remove xwiki-common from another CI: 
https://github.com/checkstyle/checkstyle/blob/32cb8fc7aea1595c175c543a370b4c06fb41280d/.ci/wercker.sh#L97
https://github.com/checkstyle/checkstyle/blob/32cb8fc7aea1595c175c543a370b4c06fb41280d/wercker.yml#L86